README.md

```diff
@@ -1124,13 +1124,25 @@ condition](#safe-assignment-in-condition).
   # good
   names.each { |name| puts name }
 
+  Use this proc invocation shorthand when the invoked method / attr is the only operation of a block
+
+  # bad
+  names.map { |name| name.upcase }
+
+  # good
+  names.map(&:upcase)
+  ```
+
+* <a name="block-chaining">Chaining</a>
```

Comment:
I'm still confused by the change you made here. Using the shorthand syntax should have been a separate rule.